optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/SingleStatistic.java

```diff
@@ -178,6 +178,18 @@ public void readCsvStatisticFile() {
         }
     }
 
+    public void retrievePointList() {
+        if (getCsvFile().exists() && pointList == null) {
```

Comment:
Definitely fail fast if the csv file doesn't exist. Suppose I remove the CSV file manually during benchmark and this method doesn't crash - then I get some obscure NPE later, instead of a nice IllegalStateException telling me about the missing CSV file.

I would fail fast if the pointList isn't null. In theory this method is never called when pointList isn't null, and the rest of our coding presumes that. So because we haven't actively dealt with the situation of that happening, we should fail fast so it's sticks out like sore thumb when it does happen (which I presume will be never).

Suppose the CVS doesn't exist -